openml/tasks/functions.py

```diff
@@ -235,7 +236,7 @@ def get_task(task_id):
 
     # TODO look into either adding the class labels to task xml, or other
     # way of reading it.
-    class_labels = dataset._retrieve_class_labels()
+    class_labels = dataset._retrieve_class_labels(task.target_feature)
```

Comment:
This is actually an upstream issue: http://www.openml.org/api_docs/#!/task/get_task_id

Not sure now if I should fix this.